src/SourceTerms/Leakage/CPU_Src_Leakage_ComputeLeak.cpp

```diff
@@ -0,0 +1,1154 @@
+#include "CUFLU.h"
+#include "NuclearEoS.h"
+#include "Src_Leakage.h"
+
+
+
+#ifndef __CUDACC__
+
+extern bool   IsInit_dEdt_Nu     [NLEVEL];
+extern double CCSN_Leakage_EAve  [NType_Neutrino];
+extern double CCSN_Leakage_RadNS [NType_Neutrino];
```

Comment:
Any reason to have a empty space before `[]`

Reply:
Removed